packages/runtime/test/transport/identityDeletionPeerProcessing.test.ts

```diff
@@ -1,22 +1,27 @@
 import { IdentityDeletionProcessStatus } from "@nmshd/transport";
 import { PeerDeletionCancelledEvent, PeerDeletionStatus, PeerToBeDeletedEvent } from "../../src";
-import { establishRelationship, RuntimeServiceProvider, syncUntilHasEvent, TestRuntimeServices } from "../lib";
+import { establishRelationship, exchangeMessageWithRequest, RuntimeServiceProvider, sendMessageToMultipleRecipients, syncUntilHasEvent, TestRuntimeServices } from "../lib";
 
 const serviceProvider = new RuntimeServiceProvider();
 let services1: TestRuntimeServices;
 let services2: TestRuntimeServices;
+let services3: TestRuntimeServices;
 let relationshipId: string;
+let relationshipId2: string;
 
 beforeAll(async () => {
-    const runtimeServices = await serviceProvider.launch(2);
+    const runtimeServices = await serviceProvider.launch(3, { enableRequestModule: true, enableDeciderModule: true, enableNotificationModule: true });
```

Comment:
We should be very careful with enabling modules for test suites. Is there no other possibility?